### DISCUSSION TITLE
LES simulation

### DISCUSSION THREAD

**Discussion by owen14121022** [CliMA/Oceananigans.jl#1356] [Computational science!] [answered]
In the Langmuir Simulation example, are the resolved velocity fields (u,v,w) the Lagrangian-mean velocity fields?

**Comment by ali-ramadhan** [comment, marked as answer]
Yes I believe so but I'll let @glwagner confirm since he wrote that example and the `SurfaceWaves` module (now called `StokesDrift` as of yesterday).

He also wrote this in the example:

> In other words, `model.velocities.u` is the Lagrangian-mean _x_-velocity beneath surface waves.

**Reply by navidcy** [reply]
@owen14121022 I'm also pretty sure the answer to your questions is *yes*. See also here: https://clima.github.io/OceananigansDocumentation/stable/physics/surface_gravity_waves/

**Reply by glwagner** [reply]
Yes, `model.velocities.u`, etc, are the Lagrangian-mean velocities when `surface_waves` / `stokes_drift` is used.

**Reply by owen14121022** [reply]
> Yes, `model.velocities.u`, etc, are the Lagrangian-mean velocities when `surface_waves` / `stokes_drift` is used.

Thank you for your reply！Have you compared the model result (for example，horizontal and time-mean horizontal velocity profiles) with that in the McWilliams 1997.

**Reply by glwagner** [reply]
I did that in a [paper (not published, but in press for JPO)](https://glwagner.github.io/assets/pdf/near-inertial-waves-turbulence-growth-swell-preprint.pdf) for a high-resolution version of the example.

<img width="1313" alt="image" src="https://user-images.githubusercontent.com/15271942/108604528-7621d600-737c-11eb-8832-964483e00b5a.png">

This is figure 2 from McWilliams et al. 1997:

<img width="1069" alt="image" src="https://user-images.githubusercontent.com/15271942/108604542-8f2a8700-737c-11eb-8fa0-b141e5dea723.png">

and figure 6:

<img width="1050" alt="image" src="https://user-images.githubusercontent.com/15271942/108604552-9ce00c80-737c-11eb-9a51-1fae94271e9e.png">

My preprint contains some discussion (as well as demonstrating via its analysis) why the Lagrangian-mean velocity field is a more appropriate and physically-intuitive prognostic variable than the Eulerian-mean velocity field.

In appendix A, I also argue that using the Lagrangian-mean rate of strain in turbulence closures for LES is more appropriate for modeling downscale transfers of kinetic energy than a closure formulated in terms of the Eulerian-mean rate of strain --- because closures formulated in terms of the Eulerian-mean rate of strain do not always dissipate mean kinetic energy.
